source/velocity_boundary_conditions/interface.cc

```diff
@@ -70,7 +70,8 @@ namespace aspect
     }
 
 
-
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
```

Comment:
Let's at least silence this for the common case (GCC), release deal.II, release ASPECT, and then just remove the whole shebang again.